### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by domenicsim1**
Moved to net5.0
+semver: minor

### CODE REVIEW

File: build/Octopus.Server.Extensibility.Authentication.nuspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (slewis74) [inline build/Octopus.Server.Extensibility.Authentication.nuspec]: Per the changes for Data, we should be able to package this just using `DotNetPublish`

**Review by slewis74 — CHANGES_REQUESTED** [review]
One change to drop the nuspec

File: build/Octopus.Server.Extensibility.Authentication.nuspec (hunk 1 above)
Comment (domenicsim1) [reply build/Octopus.Server.Extensibility.Authentication.nuspec]: fixed

**Review by slewis74 — APPROVED** [review]
👍

**Pull request merged**